wolfcrypt/src/asn.c

```diff
@@ -8915,7 +8915,7 @@ int ParseCertRelative(DecodedCert* cert, int type, int verify, void* cm)
                     if (ret != 0 && ret != WC_PENDING_E) {
                         WOLFSSL_MSG("Confirm signature failed");
                     }
-                    return ret;
+                    goto exit_pcr;
```

Comment:
This doesn't look right to me.  In the non async case imagine a date error and then confirm signature fails with ASN_SIG_CONFIRM_E.  Now we're jumping to return a bad date error (and ignoring the more important signature error).  What if the user overrides a date error from a bad clock and doesn't realize the signature failed?

Reply:
Fixed so `ParseCertRelative` exits earlier in the case where a date error is a verify failure.